backend/testUtils/testSession.ts

```diff
@@ -185,6 +185,47 @@ export const mockTestSessions: TestSessionRequestDTO[] = [
   },
 ];
 
+export const mockTestSessionsWithEvenNumberOfResults: TestSessionRequestDTO[] = [
+  {
+    test: mockTestWithId.id,
+    teacher: mockTeacher.id,
+    school: mockSchoolWithId.id,
+    gradeLevel: 4,
+    results: [
+      mockGradedTestResult,
+      mockGradedTestResult2,
+      mockGradedTestResult3,
+      mockUngradedTestResult,
+      mockGradedTestResult,
+      mockGradedTestResult4,
+    ],
+    accessCode: "1234",
+    startTime: new Date("2021-09-01T09:00:00.000Z"),
```

Comment:
```suggestion
    ... mockTestSessions[0],
    results: [
      mockGradedTestResult,
      mockGradedTestResult2,
      mockGradedTestResult3,
      mockUngradedTestResult,
      mockGradedTestResult,
      mockGradedTestResult4,
    ],
```
can we refactor these

Reply:
Done!